nilearn/image/image.py

```diff
@@ -665,3 +665,68 @@ def threshold_img(img, threshold, mask_img=None):
     threshold_img = new_img_like(img, img_data, affine)
 
     return threshold_img
+
+
+def math_img(formula, **imgs):
+    """Interpret a string formula using niimg in named parameters.
```

Comment:
> General comment: Any formula in string works ? such as "sum" "div" or It should be in numpy context ?

For me moment, the function assumes the input data are Nifti image, thus they contains numpy arrays. So normally, the formula should contains any function that can be applied on numpy arrays.